### PR TITLE
Authenticate customer docs

### PR DESCRIPTION
**Pull request by iurevych**
Docs rework:
1. Integration process for SaaS and Custom Platform is broken down into to pieces now (inspired by Google Analytics docs): a) place the lib inside head along with `authenticate_customer`, b) show different campaigns right after
2. `customer` subhash documentation
3. SaaS doc explains how to show different campaigns now not only PP
4. Container ID fixed to match new 2.0 integration lib

### CODE REVIEW

**Comment by iurevych** [comment]
@zhuravel I need your eyes on it, please review. Thanks.

File: source/campaigns/site_placements/invite.rst [hunk 1]
```diff
@@ -30,16 +33,30 @@ Once you have created the page below is an example of the basic Invite campaign
...
-.. note::
+  <!-- Place Talkable Container into appropriate place in the DOM -->
+  <div id="talkable-invite"></div>
+
+  <script>
+    var _talkable_data = {
+      // campaign_template: {
+      // ...
+      // },
+      customer {
```
Comment (zhuravel) [inline source/campaigns/site_placements/invite.rst]: Missing colon, should be `customer: {`

File: source/campaigns/site_placements/invite.rst [hunk 2]
```diff
@@ -30,16 +33,30 @@ Once you have created the page below is an example of the basic Invite campaign
...
+        email 'overridden@example.com',
```
Comment (zhuravel) [inline source/campaigns/site_placements/invite.rst]: Missing colon, should be `email: 'overridden@example.com',`

File: source/campaigns/site_placements/post_purchase.rst [hunk 3]
```diff
@@ -77,23 +76,44 @@ Post Purchase campaign can be also embedded as inline widget somewhere on the pa
...
+
+  <!-- Place Talkable Container into appropriate place in the DOM -->
+  <div id="talkable-post-purchase"></div>
+
+  <script>
+    var _talkable_data = {
+      // purchase: {
       // ...
+      // },
+      customer {
```
Comment (zhuravel) [inline source/campaigns/site_placements/post_purchase.rst]: Same here: `customer: {`, `email: {`

**Comment by iurevych** [comment]
Thanks!

**Comment by zhuravel** [comment]
Looks good

**Pull request merged**